src/ChorusMerge.Tests/ChorusMergeTests.cs

```diff
@@ -77,14 +76,9 @@ public void Main_Utf8FilePaths_FileNamesOk()
 				var filePath3 = Path.Combine(e.Path, "aaa.chorusTest");
 				File.WriteAllText(filePath3, @"aaa");
 
-				var encoding = Encoding.GetEncoding(1252);
```

Comment:
The test is actually using file paths with non-Roman characters: there's "ไก่ projéct" in the project path. But I'll go ahead and replace those filenames with non-Roman characters as well. I'll even make sure to include emojis like 📚 (U+1F4DA) to make sure that surrogate pairs are being correctly handled as well. But since [Python 3.6 revamped filesystem handling](https://peps.python.org/pep-0529/) to use UTF-8 everywhere internally, and convert that to UTF-16 before passing filenames to Windows, I expect it to Just Work™. I'll let you know if that change breaks anything in the test.